src/test/java/LIC13Test.java

```diff
@@ -0,0 +1,94 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+final public class LIC13Test {
+  Parameters params = new Parameters();
+  
+  
+  @Test
+    /*
+    * It tests that LIC13 returns false when the parameters are incorrect
+  **/
+
+  public void testLIC13Inputs(){
+    Parameters params = new Parameters();
+    params.RADIUS2 = -1;
+
+    Point[] dp = new Point[] { new Point(1, 1), new Point(1, 1), new Point(1, 1), new Point(1, 1), new Point(1, 1) };
+    CMV cmv = new CMV(params, dp);
+    cmv.populate();
+
+    assertFalse(cmv.get()[13]);
+    
+    //Test points input
+    params.RADIUS2 = 10;
+    dp = new Point[] { new Point(1, 1), new Point(1, 1)};
+    cmv = new CMV(params, dp);
+
+    assertFalse(cmv.get()[13]);
+
+  }
+
+  @Test
+  /*
+  * It tests that LIC13 is True when the correct conditions are met
+  */
+  public void testLIC13True(){
+
+    Parameters params = new Parameters();
```

Comment:
Since there was a minor error in the LIC13-function (looking at <= RADIUS1 instead of > RADIUS1), this test will probably fail when that is fixed. Or at least I can't see three points that couldn't be encompassed by a circle of radius 1?